internal/sql/sql.go

```diff
@@ -38,6 +38,7 @@ type (
 		log               *zap.Logger
 		longQueryDuration time.Duration
 		longTxDuration    time.Duration
+		partialSlabDir    string
```

Comment:
This doesn't belong in here. I'd pass it in via the `Migrator` interface rather than introducing it all the way down in our SQL driver wrapper.